src/mdbExtensionController.ts

```diff
@@ -487,6 +487,38 @@ export default class MDBExtensionController implements vscode.Disposable {
         return Promise.resolve(true);
       }
     );
+    this.registerCommand(
+      EXTENSION_COMMANDS.MDB_COPY_DOCUMENT_ID_FROM_TREE,
+      async (element: DocumentTreeItem): Promise<boolean> => {
+        const { documentId } = element;
+
+        if (!documentId) {
+          void vscode.window.showWarningMessage('Can\'t copy ID from document that has no ID.');
+          return false;
+        }
+
+        let body = JSON.stringify(documentId);
+
+        if (body.startsWith('"')) {
+          body = body.slice(1, body.length - 1);
+        }
```

Comment:
If the idea here is to leave string document ids as-is, it might be better to check e.g. `if (typeof body !== 'string') { body = JSON.stringify(documentId); }` instead, since this currently still leaves JSON escapes in `body`